helm_deploy/values-dev.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: quay.io/hmpps/hmpps-interventions-ui
+  tag: latest
+  pullPolicy: IfNotPresent
+  port: 3000
+
+ingress:
+  enabled: true
+  enable_whitelist: true
+  host: hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk
```

Comment:
i changed this to match the change @sldblog made to hmpps-interventions-service. 

we need to track the fact that this needs changing when we have our real urls set up